src/server/scripts/Outland/CoilfangReservoir/SerpentShrine/instance_serpent_shrine.cpp

```diff
@@ -101,8 +101,8 @@ class instance_serpent_shrine : public InstanceMapScript
             {
                 case NPC_COILFANG_SHATTERER:
                 case NPC_COILFANG_PRIESTESS:
-                    if (creature->GetPositionX() > -110.0f && creature->GetPositionX() < 155.0f && creature->GetPositionY() > -610.0f && creature->GetPositionY() < -280.0f)
-                        AliveKeepersCount += creature->IsAlive() ? 0 : -1; // SmartAI calls JUST_RESPAWNED in AIInit...
+                    if (creature->GetPositionX() > 190.0f)
+                        --AliveKeepersCount;
```

Comment:
is it okay to use on the outer side of a line here instead of outside a box? I haven't checked

Reply:
Clean up the count outside the "circular platform"